man/rc-sstat.8

```diff
@@ -20,7 +20,7 @@ info
 .Sh DESCRIPTION
 .Nm
 gathers and displays information about the status of services supervised
-by s6 then runs rc-status to show info about nnormal OpenRC services.
+by s6 then runs rc-status to show info about normal OpenRC services.
```

Comment:
Sometimes random letters are intentional, but I don't _think_ that's the case here.